modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/tabletoolbar/plugin.js

```diff
@@ -0,0 +1,1475 @@
+(function () {
+	function getSelectedCells(selection) {
+		const ranges = selection.getRanges();
+		const retval = [];
+		const database = {};
+
+		function moveOutOfCellGuard(node) {
+			// Apply to the first cell only.
+
+			if (retval.length > 0) {
+				return;
+			}
+
+			// If we are exiting from the first </td>, then the td should definitely be
+			// included.
```

Comment:
Minor SF: We should remove all comments.

Reply:
Some comments may be useful, isn't? I would prefer to maintain

Reply:
@diegonvs  Generally, we are making great effort in Liferay that code should be self-explanatory. Examples that achieve this are meaningful variable and function names, simplicity and limited scope of what a function / class does. Inline comment is only useful when the line of code does not make sense from the close context, or we want to ensure future devs don't change something that looks innocent but is critical to leave as it is. 

Examples of good inline comments:
-  this [rendering of component seemingly nowhere](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js#L264-L267)
-  [this setting of id to a very specific value](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js#L76-L80). 

I don't think any of comments in this PR fall in these categories, there are many that just explain algorithm flow or variable names. There might be valid ones, I didn't go deep in analysis. But, inline commnets should be an exception, with a very clear reason, then considered and approved in a review.

Reply:
Thanks for providing this detailed answer :)

Yes, I found some comments that can be just removed or I can improve variable naming to make the code self-explanatory. I'll change it.